### ISSUE TITLE
Yum packages for Fedora

### ISSUE THREAD

**Issue by mattboston** [furlongm/patchman#276]
Would it be possible to create rpms for Fedora?  I get the following error when I try to install.  If I use --skip-broken, it installs the client, but not the server. The requisite packages that Fedora has are python3-progressbar and python3-magic

```
[root@nasbox ~]# yum install -y patchman patchman-client
Last metadata expiration check: 2:04:34 ago on Mon 11 Jan 2021 09:16:59 AM EST.
Error: 
 Problem: conflicting requests
  - nothing provides python-progressbar needed by patchman-1.0.17-1.noarch
  - nothing provides python-magic needed by patchman-1.0.17-1.noarch
  - nothing provides python-progressbar needed by patchman-1.0.18-1.noarch
  - nothing provides python-magic needed by patchman-1.0.18-1.noarch
  - nothing provides python-progressbar needed by patchman-1.0.19-1.noarch
  - nothing provides python-magic needed by patchman-1.0.19-1.noarch
  - nothing provides python-progressbar needed by patchman-1.0.20-1.noarch
  - nothing provides python-magic needed by patchman-1.0.20-1.noarch
  - nothing provides python-progressbar needed by patchman-1.0.21-1.noarch
  - nothing provides python-magic needed by patchman-1.0.21-1.noarch
  - nothing provides python-progressbar needed by patchman-1.0.22-1.noarch
  - nothing provides python-magic needed by patchman-1.0.22-1.noarch
  - nothing provides python-progressbar needed by patchman-1.2.0-1.noarch
  - nothing provides python-magic needed by patchman-1.2.0-1.noarch
  - nothing provides python-progressbar needed by patchman-1.2.0-2.noarch
  - nothing provides python-magic needed by patchman-1.2.0-2.noarch
  - nothing provides python-progressbar needed by patchman-1.2.1-1.noarch
  - nothing provides python-magic needed by patchman-1.2.1-1.noarch
  - nothing provides python-progressbar needed by patchman-1.2.12-1.noarch
  - nothing provides python-magic needed by patchman-1.2.12-1.noarch
  - nothing provides python-progressbar needed by patchman-1.2.2-1.noarch
  - nothing provides python-magic needed by patchman-1.2.2-1.noarch
  - nothing provides python-progressbar needed by patchman-1.2.5-1.noarch
  - nothing provides python-magic needed by patchman-1.2.5-1.noarch
(try to add '--skip-broken' to skip uninstallable packages)```

**Comment by furlongm** [comment]
This should be fixed with the 2.0 release. Check out the `develop` branch if you want to play with that. rpm build instructions are in BUILD.md

**Comment by furlongm** [comment]
The v2 packages and installation instructions are now released :-) Let me know if you have any issues.
